recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "openmp" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "db55d85a7bb289804dc42fc5c8e35ca24dfc3885782261b675a194fd7e206e26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://releases.llvm.org/{{ version }}/openmp-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: build.sh
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - llvmdev
+    - cmake
+    - perl
```

Comment:
Please pin to `5.22.2.1`. Otherwise `conda-build` will create a matrix of `perl` versions, which is unneeded here.